content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -282,7 +282,7 @@ below:
 
 {{< feature-state state="beta" for_k8s_version="v1.23" >}}
 
-GetAllocatableResources provides information on resources initially available on the worker node.
+GetAllocatableResources provides information on resources initially available on the node.
```

Comment:
generic changes from "worker node" to "node" seem fine but it depends on context